app/lib/features/settings/widgets/session_card.dart

```diff
@@ -59,7 +55,7 @@ class SessionCard extends ConsumerWidget {
                   Padding(
                     padding: const EdgeInsets.symmetric(horizontal: 5),
                     child: Text(
-                      L10n.of(ctx).logOut,
```

Comment:
Why is this being unlocalised ? We are replacing all static strings with localisation...

Reply:
Localization for cross signing contained several parametric sentences, like `Waiting for %s…`.
So I introduced `sprintf` package for them before.
But now `arb` for l10n is not used much in acter and we are almost using english sentence as UI text directly.
I thought it is better to remove `sprintf` from dependencies.

Reply:
When needed, I will restore them to l10n using appropriate manner.

Reply:
Will fix it in [another PR](https://github.com/acterglobal/a3/pull/1539).